hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -1450,4 +1450,27 @@ public OzoneManagerProtocol getOzoneManagerClient() {
   public Cache<URI, KeyProvider> getKeyProviderCache() {
     return keyProviderCache;
   }
+
+  @Override
+  public OzoneKey headObject(String volumeName, String bucketName,
+      String keyName) throws IOException {
+    Preconditions.checkNotNull(volumeName);
+    Preconditions.checkNotNull(bucketName);
+    Preconditions.checkNotNull(keyName);
+    OmKeyArgs keyArgs = new OmKeyArgs.Builder()
+        .setVolumeName(volumeName)
+        .setBucketName(bucketName)
+        .setKeyName(keyName)
+        .setRefreshPipeline(false)
+        .setSortDatanodesInPipeline(false)
```

Comment:
This should be a no-op as we skip it in OM if `headOp` flag is set, we can skip it here?

Reply:
Done

Reply:
Done